modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_paragraph.scss

```diff
@@ -4,8 +4,8 @@
 
   p {
     @include big-paragraph;
-    @include modular-spacing('margin-bottom', 1);
-    @include margin(0 null null null);
+    @include margin(0 null 0 null);
+    @include modular-spacing('padding-bottom', 2);
```

Comment:
This is mixing padding and margin again. Shouldn't we stick to one to be more predictable?

Reply:
That's a great question. I was following what @kerri-augenstein  had specified in this ticket. 
https://stanfordits.atlassian.net/browse/D8CORE-1401

Reply:
We are using margin everywhere else. Stick to using margin on this one, please.

Reply:
@sherakama in the ticket https://stanfordits.atlassian.net/browse/D8CORE-1401, Kerri has specifically asked to use margin.

> I think we want to kill the margin applied to our <p>, and instead opt for padding only, as the margin on the <p> is overlapping with the margins applied to headings, and we want them to stack.

Reply:
Please stick to using margin only. Both padding and margin are accomplishing the same thing in this case and I'd rather stay consistent with our other elements.